packages/webapp/src/components/Tile/DescriptionListTile/index.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ *  Copyright 2025 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import { CSSProperties, ReactNode } from 'react';
+import clsx from 'clsx';
+import styles from './styles.module.scss';
+
+export type TileData = { label: string; data: ReactNode; iconURL?: string };
+
+export enum LabelSize {
+  SMALL = 'small',
+}
+
+interface DescriptionListTileProps extends TileData {
+  hideLabel?: boolean;
+  labelSize?: LabelSize;
+  className?: string;
+}
+
+// Parent element must be <dl> for proper semantic structure
```

Comment:
To ensure this is actually the case, I'd probably make a `DescriptionList` component instead that receives an array for the children props and renders the list items